### ISSUE TITLE
Sequential Aria Level Headers

### ISSUE THREAD

**Issue by joe-replin** [adaptlearning/adapt-contrib-core#145]
### Bug Description:

Aria level headers get skipped in numbering when article headers & block headers are not present. 

### Steps to Reproduce:

Using a screen reader, navigate through the headings, noting the heading level of each one.

### Actual Behavior:

When article & block headers are not present, heading levels will start at 1 for a page header and move to 4 for subsequent headers, with no mention of 2 or 3. Adapt currently assigns heading levels to the following elements:

- heading level 1 for page headers
- heading level 2 for article headers
- heading level 3 for block headers
- heading level 4 for component headers
- heading level 5 for component item headers

### Expected Behavior:

From https://www.w3.org/WAI/tutorials/page-structure/headings/
Skipping heading ranks can be confusing and should be avoided where possible: Make sure that a h2 is not followed directly by an h4, for example. It is ok to skip ranks when closing subsections, for instance, a h2 beginning a new section, can follow an h4 as it closes the previous section.

This would mean Adapt dynamically assigning aria heading levels to each level. A component would adopt a heading level of 2 if an article or block header did not currently exist.

**Comment by oliverfoster** [comment]
We can give it a shot, I haven't automated it because I'm never quite sure how the content is structured and whether we can make the above assumptions?
